www/src/views/showcase/filtered-showcase.js

```diff
@@ -72,31 +80,37 @@ class FilteredShowcase extends Component {
     }
 
     // create map of categories with totals
-    const aggregatedCategories = data.allSitesYaml.edges.reduce(
-      (categories, edge) => {
-        if (edge.node.categories) {
-          edge.node.categories.forEach(category => {
-            // if we already have the category recorded, increase count
-            if (categories[category]) {
-              categories[category] = categories[category] + 1
-            } else {
-              // record first encounter of category
-              categories[category] = 1
-            }
-          })
+    const aggregatedCategories = items.reduce((categories, edge) => {
+      if (!edge.node.categories) {
+        edge.node.categories = []
+      }
+      const idx = edge.node.categories.indexOf(OPEN_SOURCE_CATEGORY)
+      if (idx !== -1) {
+        edge.node.categories.splice(idx, 1)
+      }
+      if (edge.node.source_url) {
+        edge.node.categories.push(OPEN_SOURCE_CATEGORY)
```

Comment:
This is also a nice touch, adding any sites with a public repository to the Open Source category, seems like a good idea to me!

Reply:
oh yeah, great idea!